client/my-sites/sidebar-unified/static-data/jetpack-menu.js

```diff
@@ -0,0 +1,248 @@
+			children: [
+				{
+					parent: 'jetpack',
+					slug: 'jetpack-activity-log',
+					title: translate( 'Activity Log' ),
+					type: 'submenu-item',
+					url: `/activity-log/${ siteDomain }`,
+				},
+				{
+					parent: 'jetpack',
+					slug: 'jetpack-backup',
+					title: translate( 'Backup' ),
+					type: 'submenu-item',
+					url: `/backup/${ siteDomain }`,
+				},
+				{
+					parent: 'jetpack',
+					slug: 'jetpack-scan',
+					title: translate( 'Scan' ),
+					type: 'submenu-item',
+					url: `/scan/${ siteDomain }`,
+				},
+				{
+					parent: 'jetpack',
+					slug: 'jetpack-search',
+					title: translate( 'Search' ),
+					type: 'submenu-item',
+					url: `/jetpack-search/${ siteDomain }`,
+				},
+			],
+		},
+		{
+			type: 'separator',
+		},
+		{
+			icon: 'dashicons-admin-appearance',
+			slug: 'themes-php',
+			title: translate( 'Appearance' ),
+			type: 'menu-item',
+			url: `/themes/${ siteDomain }`,
+			children: [
+				{
+					parent: 'themes.php',
+					slug: 'themes-php',
+					title: translate( 'Themes' ),
+					type: 'submenu-item',
+					url: `/themes/${ siteDomain }`,
+				},
+				{
+					parent: 'themes.php',
+					slug: 'themes-customize',
+					title: translate( 'Customize' ),
+					type: 'submenu-item',
+					url: `https://${ siteDomain }/wp-admin/customize.php`,
+				},
+			],
+		},
+		{
+			icon: 'dashicons-admin-plugins',
+			slug: 'plugins',
+			title: translate( 'Plugins' ),
+			type: 'menu-item',
+			url: `/plugins/${ siteDomain }`,
+		},
+		{
+			icon: 'dashicons-admin-users',
+			slug: 'users-php',
+			title: translate( 'Users' ),
+			type: 'menu-item',
+			url: `/people/team/${ siteDomain }`,
+		},
+		{
+			icon: 'dashicons-admin-tools',
+			slug: 'tools-php',
+			title: translate( 'Tools' ),
+			type: 'menu-item',
+			url: `/marketing/tools/${ siteDomain }`,
+			children: [
+				{
+					parent: 'tools.php',
+					slug: 'tools-marketing',
+					title: translate( 'Marketing' ),
+					type: 'menu-item',
+					url: `/marketing/tools/${ siteDomain }`,
+				},
+				{
+					parent: 'tools.php',
+					slug: 'tools-earn',
+					title: translate( 'Earn' ),
+					type: 'menu-item',
+					url: `/earn/${ siteDomain }`,
+				},
+				{
+					parent: 'tools.php',
+					slug: 'tools-import',
+					title: translate( 'Import' ),
+					type: 'submenu-item',
+					url: `https://${ siteDomain }/wp-admin/import.php?calypsoify=0`,
+				},
+				{
+					parent: 'tools.php',
+					slug: 'tools-export',
+					title: translate( 'Export' ),
+					type: 'submenu-item',
+					url: `https://${ siteDomain }/wp-admin/import.php?calypsoify=0`,
+				},
+			],
+		},
+		{
+			icon: 'dashicons-admin-settings',
+			slug: 'options-general-php',
+			title: translate( 'Settings' ),
+			type: 'menu-item',
+			url: `/settings/general/${ siteDomain }`,
+		},
+		{
+			icon: WPADMIN_ICON,
+			slug: 'wp-admin',
+			title: translate( 'WP Admin' ),
+			type: 'menu-item',
+			url: `https://${ siteDomain }/wp-admin`,
+		},
```

Comment:
Should this be renamed to "Dashboard" and move it to the top, so it's better aligned with the dynamic menu?

<img width="537" alt="Screen Shot 2021-01-29 at 11 02 58" src="https://user-images.githubusercontent.com/1233880/106261209-92b96b00-6221-11eb-853e-3fafd6b6bffd.png">

Reply:
This one aligns with this comment https://github.com/Automattic/jetpack/pull/18586#issuecomment-770760314. Let's make it consistent and not let users believe that this is their WPAdmin menu.